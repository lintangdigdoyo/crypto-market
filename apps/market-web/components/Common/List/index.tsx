interface ListProps {
  children: React.ReactNode;
}

const List = ({ children }: ListProps) => {
  return (
    <div className="flex justify-between p-4 border-t border-slate-200">
      {children}
    </div>
  );
};

export default List;
