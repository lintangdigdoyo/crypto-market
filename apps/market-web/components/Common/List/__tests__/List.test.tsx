import renderer from "react-test-renderer";
import "@testing-library/jest-dom";

import List from "..";

describe("List component", () => {
  it("should renders List correctly", () => {
    const props = {
      children: <>children</>,
    };
    const component = renderer.create(<List {...props} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
