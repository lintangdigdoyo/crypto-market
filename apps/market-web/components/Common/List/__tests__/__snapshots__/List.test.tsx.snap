// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List component should renders List correctly 1`] = `
<div
  className="flex items-center justify-between p-4 border-t border-slate-200 last:border-b"
>
  children
</div>
`;
